python/test/unit/refinement/interval.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2024 Paul Kühner
+#
+# This file is part of DOLFINx (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+
+from mpi4py import MPI
+
+import numpy as np
+import pytest
+
+from dolfinx import mesh
+
+
```

Comment:
General comment: Add some tests checking the parent->child hierarchy is correct.

Reply:
In the cpp tests, where we have control over all related indices, they are present. Not to sure how to facilitate this in these highly parameterized setting.